submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/ClusterProperties.kt

```diff
@@ -11,6 +11,24 @@ data class ClusterProperties(
     val enabled: Boolean,
     val user: String,
     val key: String,
-    val server: String,
```

Comment:
Since we're executing the tasks in either one cluster or the other, I think we shouldn't change this parameter but just set the corresponding property in the config files.

Another reason that makes me think this is that, eventually, we'll get rid of LSF entirely or add another cluster implementation so we would need to keep adding them here

Reply:
I see your point but I think during transition period better two have an option for both to quickly change. During my testing of slurm infra team did find issues and server was not functionall so I suspect having both options for a while will be usefull. Once feature (and service in the EBI) stabilize I do agree better to a single one.

Reply:
I agree, let's then create a ticket for removing it in the future